src/lambda/io.rs

```diff
@@ -207,11 +216,15 @@ async fn execute_raw_grpc_request<'ctx, Ctx: ResolverContextLike<'ctx>>(
     ctx: &EvaluationContext<'ctx, Ctx>,
     req: Request,
     operation: &ProtobufOperation,
+    protobuf_set: &ProtobufSet,
 ) -> Result<Response<async_graphql::Value>> {
     Ok(
-        execute_grpc_request(&ctx.request_ctx.runtime, operation, req)
+        execute_grpc_request(&ctx.request_ctx.runtime, operation, protobuf_set, req)
             .await
-            .map_err(|e| EvaluationError::IOException(e.to_string()))?,
+            .map_err(|e| match e.downcast::<EvaluationError>() {
+                Ok(err) => err,
+                Err(err) => EvaluationError::IOException(err.to_string()),
+            })?,
```

Comment:
could `impl From<anyhow::Error> for EvaluationError` simplify error managemenet over this file so we'll have only `e.into()` here?

Reply:
done for both `anyhow::Error` and `Arc<anyhow::Error>`